notebooks/data-sources/BUSpark-CS506/RedHatNLP.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
```

Comment:
<p>Some suggestions:</p><ul><li>If there are 256 logs, training with 100 and testing with 156 may not be enough for the model to train. 80% training and 20% test with StratifiedShuffleSplit split should give better results. </li><li>Try xgboost:</li></ul><pre class="ql-syntax ql-indent-1" spellcheck="false">from xgboost import XGBClassifier
XGBClassifier().fit(X_train, y_train)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>